.github/workflows/ingest_library_compare.yml

```diff
@@ -11,19 +11,33 @@ on:
       schema_name:
```

Comment:
oh, sorry, forgot to ask about this, but one line up: `default: dcp_mappluto` do we want that?

Reply:
Eh, not really. Will remove